### PR TITLE
Do not remove image's styles

### PR DESCRIPTION
**Pull request by gela-d**
Before, code from bse-admin remove style for images
`<p></p>`
after, we left all of attributes
`<p>фотография программы с минского BEMup</p>`

### CODE REVIEW

**Comment by gela-d** [comment]
@tormozz48

File: src/tasks/override-links.js [hunk 1]
```diff
@@ -212,8 +212,7 @@ function overrideLinks(content, node, urlHash, lang, doc) {
 
     content = content.replace(REGEXP.SRC, function(str, href){
         //if href is absolute (src="http://..." ) return href
-        href = /^http/.test(href) ? href : buildSrcForImages(href, doc);
-        return '<img src="' + href + '"';
+        return /^http/.test(href) ? str : '<img src="' + buildSrcForImages(href, doc) + '"';
```
Comment (tadatuta) [inline src/tasks/override-links.js]: for relative links attrs will still be dropped, isn't it?

Comment (tavriaforever) [inline src/tasks/override-links.js]: I am agree, looking very strange

**Comment by gela-d** [comment]
@tadatuta @tavriaforever thank u, Update PR

File: src/tasks/override-links.js [hunk 2]
```diff
@@ -155,10 +155,11 @@ function recognizeRelativeLinkForLibraryDocs(str, node) {
  * @param {BaseNode} doc node
  * @returns {*}
  */
-function buildSrcForImages(href, doc) {
+function buildSrcForImages(str, href, doc) {
     var docParentUrl = doc && doc.url.replace(/\/\w*\.md/, '/'); // replace last url's part: repo/docs/a.md -> repo/docs
```
Comment (tadatuta) [inline src/tasks/override-links.js]: if there's a change that there's no `doc`, `src` will be broken

File: src/tasks/override-links.js [hunk 3]
```diff
@@ -155,10 +155,11 @@ function recognizeRelativeLinkForLibraryDocs(str, node) {
...
 
-    return href = docParentUrl + href + '?raw=true';
+    // change only src, save styles and attrs
+    return str = str.replace(/src=".+"/g, 'src="' + docParentUrl + href + '?raw=true"')
```
Comment (tadatuta) [inline src/tasks/override-links.js]: are double quotes garanteed?

File: src/tasks/override-links.js (hunk 2 above)
Comment (gela-d) [inline src/tasks/override-links.js]: yes, but we have no another way to know whole path to img.

File: src/tasks/override-links.js (hunk 3 above)
Comment (gela-d) [inline src/tasks/override-links.js]: I dont' understan question, sorry

Comment (gela-d) [inline src/tasks/override-links.js]: understand and up PR

**Comment by tadatuta** [comment]
actually `` is still possible and won't be rewritten but LGTM ;)

**Pull request merged**